misc/make-deps.sh

```diff
@@ -34,6 +34,7 @@ if [ ! -z "$YUM" ]; then
 fi
 if [ ! -z "$APT" ]; then
 	$sudo_command $APT install -y libvirt-dev || true
+	$sudo_command $APT install -y time || true
```

Comment:
Can you add the equivalent for Fedora systems as well please?

Reply:
fixed